### PR TITLE
ColorManagement: Clean up

### PR DESCRIPTION
**Pull request by WestLangley**
Follow-on to #25637.

### CODE REVIEW

File: src/math/ColorManagement.js [hunk 1]
```diff
@@ -43,19 +43,15 @@ const _vector = /*@__PURE__*/ new Vector3();
 
 function DisplayP3ToLinearSRGB( color ) {
 
-	color.convertSRGBToLinear();
-
-	_vector.set( color.r, color.g, color.b ).applyMatrix3( LINEAR_DISPLAY_P3_TO_LINEAR_SRGB );
-
-	return color.setRGB( _vector.x, _vector.y, _vector.z );
+	// Display P3 uses the sRGB transfer function
+	return color.convertSRGBToLinear().applyMatrix3( LINEAR_DISPLAY_P3_TO_LINEAR_SRGB );
```
Comment (WestLangley) [inline src/math/ColorManagement.js:47]: TBH, in a future PR, I would prefer to see `color.convertDisplayP3ToLinear()` added. As implemented, this looks like a bug -- in spite of the comment.

Comment (donmccurdy) [reply src/math/ColorManagement.js:43]: I'd prefer to wait and see here... I'm concerned `convertXToY` may get cluttered. We could also consider...

```js
ColorManagement.convert( color, DisplayP3ColorSpace, LinearSRGBColorSpace );
```

...to be a public-facing API, perhaps.

File: src/math/ColorManagement.js [hunk 2]
```diff
@@ -14,14 +13,13 @@ export function LinearToSRGB( c ) {
...
 
-
 /**
  * Matrices converting P3 <-> Rec. 709 primaries, without gamut mapping
  * or clipping. Based on W3C specifications for sRGB and Display P3,
  * and ICC specifications for the D50 connection space. Values in/out
  * are _linear_ sRGB and _linear_ Display P3.
  *
- * Note that both sRGB and Display P3 use the sRGB transfer functions.
+ * Note that both sRGB and Display P3 use the sRGB transfer function.
```
Comment (donmccurdy) [inline src/math/ColorManagement.js:22]: nit: this was intentional, I believe it's standard to refer to transfer functions (to/from X) as plural. I've tried to be consistent about that [in the docs](https://threejs.org/docs/#manual/en/introduction/Color-management) as well.

Comment (WestLangley) [reply src/math/ColorManagement.js:22]: I think there is only one transfer function. The other is the inverse transfer function.

File: src/math/ColorManagement.js (hunk 1 above)
Comment (WestLangley) [reply src/math/ColorManagement.js:43]: OK. :-)

File: src/math/ColorManagement.js (hunk 2 above)
Comment (WestLangley) [reply src/math/ColorManagement.js]: I think there is only one transfer function. The other is the inverse transfer function.

But I'll change it.

File: src/math/ColorManagement.js (hunk 1 above)
Comment (donmccurdy) [reply src/math/ColorManagement.js:43]: (I agree it was and is confusing to read! 😅)

**Review by donmccurdy — APPROVED** [review]

**Pull request merged**